src/TuDa.CIMS.Api/ConfigureServices.cs

```diff
@@ -1,34 +1,18 @@
-﻿using Microsoft.EntityFrameworkCore;
-using Scalar.AspNetCore;
-using TuDa.CIMS.Api.Database;
+﻿using Scalar.AspNetCore;
 
 namespace TuDa.CIMS.Api;
 
 public static class ConfigureServices
 {
-    /// <summary>
-    /// Configures the DbContext with the connection string.
-    /// </summary>
-    /// <param name="services"></param>
-    /// <param name="configuration"></param>
-    /// <returns></returns>
-    public static IServiceCollection ConfigureDbContext(this IServiceCollection services, IConfiguration configuration)
-    {
-        services.AddDbContext<CIMSDbContext>(options =>
-        {
-            options.UseNpgsql(configuration.GetConnectionString("CIMS"));
-        });
-        return services;
-    }
-
```

Comment:
Why do you have to remove this?

Reply:
We don't require this functionality since `Aspire.Npgsql.EntityFrameworkCore.PostgreSQL` offers a similar yet more robust implementation. The method `AddNpgsqlDbContext<DbContext>("...")` not only replicates the behavior of this method, but also seamlessly integrates with [Aspire's service discovery](https://learn.microsoft.com/en-us/dotnet/aspire/service-discovery/overview), providing enhanced capabilities.